src/lib/Microsoft.Health.Fhir.Ingest.Template/TemplateError.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Ingest.Template
+{
+    public class TemplateError
+    {
+        private readonly string _message;
+
+        public TemplateError(string message)
+        {
+            _message = message;
+        }
+
+        public string Message
```

Comment:
nit: You can actually remove the _message field and just use:

public string Message { get; }

You can set the Message property in the constructor. I saw this in a few other places, feel free to update if you like.

Reply:
Thanks for the suggestion. I think that could be cleaner. I'll update